### PR TITLE
extra #32 eda.ru

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdGuardExtra/issues/32
https://github.com/AdguardTeam/AdguardFilters/issues/41475

### CODE REVIEW

**Review by Alex-302 — APPROVED** [review]

File: RussianFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -238,6 +238,7 @@ nastup.net#?#.l-content > section.color_custom:has(> div > div.offset_small > di
 softportal.com#?#.relap-default-wrapper div[data-relap-max-column-quantity] > div.js-relap__item:has(> div.js-relap__item-inner > a > div.relap-default__item__label:contains(Реклама))
 overclockers.ru#?#div[itemprop="articleBody"] > h4:contains(реклама)
 vplate.ru#?#.article > div[style]:has(> ins.adsbygoogle)
+eda.ru#?#[style]:has(+ .js-advertizing__observer)
```
Comment (ameshkov) [inline RussianFilter/sections/css_extended.txt]: check it's performance plz

also looks too wide, what if it causes false positives?

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/css_extended.txt (hunk 1 above)
Comment (slavaleleka) [reply RussianFilter/sections/css_extended.txt]: I suggest to use such rules
`eda.ru##body > [style*="float"][style*="width"]:not([class]):not([id])`
`eda.ru##section.main-content > [style*="float"][style*="width"][style*="flex-direction"]:not([id])`

**Review by slavaleleka — CHANGES_REQUESTED** [review]
and these ones can be blocked by 
`eda.ru##.layout__content > div[data-vr-zone="journal"] > :not([data-href])`
`eda.ru##.column_pad > :not([data-href]):not([href])`

<details>
  <summary>yandex ads</summary>

Screenshot 2019-10-04 at 14 02 41

</details>

File: RussianFilter/sections/css_extended.txt (hunk 1 above)
Comment (Alex-302) [reply RussianFilter/sections/css_extended.txt]: too many `:not()`

**Review by zloyden — COMMENTED** [review]
I guess for the direct we can try that scriptlet `eda.ru#%#//scriptlet('abort-on-property-read', 'Object.prototype.forceRender.banner')`.

**Pull request closed without merging**